hushline/make_admin.py

```diff
@@ -4,23 +4,14 @@
 
 from hushline import create_app
 from hushline.db import db
-from hushline.model import SecondaryUsername, User
+from hushline.model import User
 
 
 def toggle_admin(username: str) -> None:
-    # First, try to find a primary user
-    user = db.session.scalars(db.select(User).filter_by(primary_username=username).limit(1)).first()
-
-    # If not found, try to find a secondary user
+    user = User.query.filter_by(primary_username=username).one_or_none()
```

Comment:
The session API supports the `one_or_none` method
```suggestion
    user = db.session.scalars(db.select(User).filter_by(primary_username=username)).one_or_none()
```